app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,27 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Workspace</title>
+  <title>Blocipedia</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <div class="container">
+    <ul class="nav nav-tabs">
+      <div class="pull-right user-info">
+        <br>
+        <P ALIGN="center">
```

Comment:
Try to use lowercase HTML tags and avoid using inline HTML styles like `ALIGN=...`. Try to keep all styles in the stylesheet files (i.e. `.scss` files).

```diff
-<P ALIGN="center">
+<p>
```